pynvim/msgpack_rpc/event_loop/asyncio.py

```diff
@@ -250,7 +282,8 @@ def _close_transport(transport):
             # Windows: for ProactorBasePipeTransport, close() doesn't take in
             # effect immediately (closing happens asynchronously inside the
             # event loop), need to wait a bit for completing graceful shutdown.
-            if os.name == 'nt' and hasattr(transport, '_sock'):
+            if (sys.version_info.major == 3 and sys.version_info.minor < 13 and
+                    os.name == 'nt' and hasattr(transport, '_sock')):
```

Comment:
```suggestion
            if sys.version_info < (3, 13) and os.name == 'nt' and hasattr(transport, '_sock')):
```

(though python 4.13 is too far in the future..)

In replying to https://github.com/neovim/pynvim/pull/584#issuecomment-2583355719: I haven't tested on Windows + python3.13, but I remember for older python versions this is still necessary.

Reply:
I added this. Thanks.